tests/test_extensions/test_ext_intersphinx.py

```diff
@@ -768,3 +768,60 @@ def test_intersphinx_cache_limit(app):
             config=app.config,
             srcdir=app.srcdir,
         )
+
+
+@pytest.mark.sphinx('dummy')
+def test_intersphinx_fetch_inventory_group_url(app):
```

Comment:
In terms of reducing test time, do we really need a full Sphinx instance to test URL normalisation?

A

Reply:
Mmh. I think I can mock app.

Reply:
There's a couple pre existing mock app classes, look at DummyApplication, FakeApp/fake_app, etc.

Reply:
I don't even need an app object and can just keep a fake Config (I still need some attributes since they are queried).